assets/js/main.js

```diff
@@ -1,172 +1,69 @@
-alert("Group 2 Project");
-// Every thiing under this point is used for the mobile responsiveness of the website 
-(function($) {
-
-	var	$window = $(window),
-		$body = $('body'),
-		$html = $('html');
 
-	// Breakpoints.
-		breakpoints({
-			large:   [ '981px',  '1680px' ],
-			medium:  [ '737px',  '980px'  ],
-			small:   [ '481px',  '736px'  ],
-			xsmall:  [ null,     '480px'  ]
-		});
 
-	// Play initial animations on page load.
-		$window.on('load', function() {
-			window.setTimeout(function() {
-				$body.removeClass('is-preload');
-			}, 100);
-		});

```

Comment:
Did you move this logic somewhere else or was it no longer needed? In general when making these types of changes you may want to add comments for why you are adding or removing logic to reduce back and forth

Reply:
It's no longer needed. I'm making sure the site s clean and loads fast!